charts/tfy-karpenter-config/templates/default/karpenter-default-ec2nodeclass.yaml

```diff
@@ -22,6 +22,8 @@ spec:
         {{ $key }}: {{ $val | quote }} 
         {{- end }}
   tags:
+    cpu: "true"
```

Comment:
can't we just use the extraTags field already present and only make the change in our env. Hardcoding the tags increases the chance of conflict with existing setup at customer's end

Reply:
another option is to use truefoundry.com/compute_type as the tag key